src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py

```diff
@@ -482,3 +486,25 @@ def import_required_arns():
             # pylint: disable=no-value-for-parameter
             _output.append(core.Fn.import_value(arn))
         return _output
+
+
+    def add_pipeline_trigger(self, trigger_type, trigger_config):
+        if trigger_type in self._accepted_triggers.keys():
```

Comment:
`in dict` always matches against the keys, so we could do this:

```suggestion
        if trigger_type in self._accepted_triggers:
```